cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -877,10 +877,13 @@ pub enum Inst {
         rn: Reg,
     },
 
-    /// Move from a GPR to a scalar FP register.
+    /// Move from a GPR to a vector register.  The scalar value is parked in the lowest lane
```

Comment:
BTW I don't mind the comment at al, but this operation is not special - virtually any instruction that operates on `S` or `D` registers (e.g. `Inst::FpuRR`) has exactly the same behaviour.